### PR TITLE
Jewels/add custom viz nr1 json guide

### PR DESCRIPTION
**Pull request by JuliaNocera**
## Description

This is the second of two new guides for the NRU programmability course.  This PR should be merged after the [first guide PR](https://github.com/newrelic/developer-website/pull/1186) which I branched off of to start this PR. 

This PR mainly adds this file `src/markdown-pages/build-apps/customize-visualizations-with-configuration.mdx`. The other file added under `/build-apps/` is from the first PR mentioned above. 

cc: @jaesius 

## Reviewer Notes

If there are links or steps needed to test this work, add them here.

## Related Issue(s) / Ticket(s)

This guide relies on a previous guide that is [currently another PR](https://github.com/newrelic/developer-website/pull/1186).

### CODE REVIEW

File: src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+
+</Intro>
+
+## Before you begin
+
+To get started, follow the <Link to="/build-apps/build-visualization">Build a custom visualization for dashboards</Link> guide to get setup with your <Link to="https://newrelic.com/signup?utm_source=developer-site">New Relic</Link> account, install the <Link to="/explore-docs/nr1-cli">New Relic One CLI</Link>, and create your first visualization. We will use the visualization you create in that guide as a starting point for this guide.
+
+## Setup the visualization component state
+
+In this first set of steps you will add component state to your visualization template form the previous guide referrenced above.
```
Comment (tyreer) [inline src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx:34]: "referrenced" -> "referenced" (sp)

File: src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+  PolarGrid,
+  PolarAngleAxis,
+  PolarRadiusAxis,
+  Treemap
+} from 'recharts';
+```
+    
+  </Step>
+
+To learn more about `Treemap` chart, go to the <Link to="https://recharts.org/en-US/examples/SimpleTreemap">Recharts Treemap example</Link> or the <Link to="https://recharts.org/en-US/api/Treemap">Recharts Treemap API docs</Link>.
```
Comment (tyreer) [inline src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx:240]: "`Treemap` chart" -> "`Treemap` charts" (plural)

File: src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+  dataKey="value"
+  ratio={4 / 3}
+  stroke="#fff"
+  fill="#8884d8"
+/>
+```
+
+  </Step>
+
+Now <Link to="/build-apps/build-visualization#render-the-visualization-in-local-development">render the visualziation in local development</Link> to see the new Treemap chart below your Radar chart.
```
Comment (tyreer) [inline src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx:260]: visualziation -> visualization (sp)

File: src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+} from 'recharts';
+```
+    
+  </Step>
+
+To learn more about `Treemap` chart, go to the <Link to="https://recharts.org/en-US/examples/SimpleTreemap">Recharts Treemap example</Link> or the <Link to="https://recharts.org/en-US/api/Treemap">Recharts Treemap API docs</Link>.
+
+  <Step>
+
+Under the `RadarChart` closing tag in render, add the `Treemap` component and the neccessary props.
```
Comment (tyreer) [inline src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx:244]: neccessary -> necessary (sp)

**Review by alexronquillo — APPROVED** [review]

**Pull request merged**